src/actions/tags.js

```diff
@@ -0,0 +1,77 @@
+// Local constants
+const PILLAR_HOST = 'http://localhost:8080/';
+const API_PREFIX = 'api/';
+
+// Exported constants
+export const TAG_REQUEST_STARTED = 'TAG_REQUEST_STARTED';
+export const TAG_REQUEST_SUCCESS = 'TAG_REQUEST_SUCCESS';
+export const TAG_REQUEST_FAILURE = 'TAG_REQUEST_FAILURE';
+
+var getInit = (body, method) => {
+  
+  var headers = new Headers({ 'Accept': 'application/json', 'Content-Type': 'application/json' });
+  var init = {
+    method: method || 'POST',
+    headers: headers,
+    mode: 'cors'
+  };
+
+  if (body) {
+    init.body = JSON.stringify(body);
+  }
+
+  return init;
+};
+
+// Functions
+export const getTags = () => {
+  return (dispatch) => {
+    dispatch(tagRequestStarted());
+    fetch(PILLAR_HOST + API_PREFIX + 'tags', getInit(null, 'GET'))
```

Comment:
since we're loading the configuration with `config.json` now, please replace isntances of `PILLAR_HOST` with the global variable `window.pillarHost`